frontend.html

```diff
@@ -74,18 +73,18 @@
         </md-button>
       </div>
       <div id="content">
-        <tab-nav></tab-nav>
+        <left-nav></left-nav>
```

Comment:
still need the tab nav here too, tab-nav is probably better called sub-nav

Reply:
Hmm, that might mess up the flexing.